README.rst

```diff
@@ -19,33 +19,75 @@ To install from PyPi
 
     $ pip install signalfx_lambda
 
+Auto-Instrumentation
+~~~~~~~~~~~~~~~~~~~~
+
+If the Python auto-instrumentation libraries are available to the wrapper, the
+Lambda handler will be auto-instrumented.
+
+To install the supported instrumentation libraries:
+
+::
+
+    $ pip install signalfx-tracing
+
+    $ sfx-py-trace-bootstrap -t <deployment_package_path>
+
+For more information, check out the full :new-page:`instructions <https://github.com/signalfx/signalfx-python-tracing#installation-and-configuration>`.
+
```

Comment:
This seems a bit early to detail this.  Maybe after the trace decorator is described?

Reply:
Removing this until some sort of `flush()` implementation in the tracer.